### ISSUE TITLE
Schnorr signature support

### ISSUE THREAD

**Issue by dinstein** [richardkiss/pycoin#366]
Do we have a plan to support schnorr signature for Bitcoin?

 [BIP340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki)

**Comment by yuval-curv** [comment]
This would be very helpful, together with [BIP341](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki) and [BIP342](https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki) (at least the key path)

**Comment by richardkiss** [comment]
I agree that it would be useful, as it seems likely this will be a significant part of bitcoin's future. I haven't had much time lately to research new things, so I can't commit to a time frame.
